main.tf

```diff
@@ -504,18 +504,19 @@ module "aci_tenant" {
   source  = "netascode/tenant/aci"
   version = "0.1.0"
 
-  count       = lookup(local.modules, "aci_tenant", true) == false ? 0 : 1
+  count       = lookup(local.modules, "aci_tenant", true) == false && local.tenant.managed == false ? 0 : 1
   name        = local.tenant.name
   alias       = lookup(local.tenant, "alias", "")
   description = lookup(local.tenant, "description", "")
 }
 
+
 module "aci_vrf" {
   source  = "netascode/vrf/aci"
   version = "0.1.3"
 
   for_each                               = { for vrf in lookup(local.tenant, "vrfs", []) : vrf.name => vrf if lookup(local.modules, "aci_vrf", true) }
-  tenant                                 = module.aci_tenant[0].name
+  tenant                                 = local.tenant.name
```

Comment:
If we remove the implicit dependency we should instead add it as an explicit dependency:
```
depends_on = [
  module.aci_tenant,
]
```